apps/Discord/uninstall

```diff
@@ -6,6 +6,4 @@ function error {
   echo -e "\\e[91m$1\\e[39m"
   exit 1
 }
-
-gio trash ~/discord-electron
-rm -f ~/.local/share/applications/discord.desktop
```

Comment:
You might want to leave that for compatibility reasons (users with the old Discord installation might need that when they want to remove it).

Reply:
good idea

Reply:
there, did that